feat: :lipstick: Added layouts "Registration Form" and "Confirmation …

Daniel-Garcia-C opened the pull request:
…Task"

**Registration Form (Custom Object)**
 - Added layout with all required fields. 
 - Added “Submit for Approval” button.
 
 **Confirmation Task (Custom Object)** 
- Added two sections: General and Registration Form.
- Added all fields to the layout. 
- Added “Submit Survey” button.

Daniel-Garcia-C committed:
feat: :lipstick: Added layouts "Registration Form" and "Confirmation Task"

Registration Form (Custom Object) - Added layout with all required fields. - Added “Submit for Approval” button Confirmation Task (Custom Object) - Added two sections: General and Registration Form - Added all fields to the layout. - Added “Submit Survey” button.

JonatanJSM commented:
You need to apply the layout so that users can see the buttons. Users would still see the old layouts. @Daniel-Garcia-C 


![image](https://github.com/user-attachments/assets/d04cc5ee-ba26-4df8-9ee2-2fc055eb18e4)



I recommend that you log in as a user with that role so that you can review them.
![image](https://github.com/user-attachments/assets/a9a74a38-7355-4783-8645-3a71edc72cc0)

Daniel-Garcia-C committed:
Merge branch 'dev' of https://github.com/JonatanJSM/Event-Management-System-for-University-Outreach into Section04/EMS-003

Daniel-Garcia-C committed:
Merge branch 'dev' of https://github.com/JonatanJSM/Event-Management-System-for-University-Outreach into Section04/EMS-003

Daniel-Garcia-C committed:
feat: :lipstick: Modify layouts Confirmation Task and Registration Form

Added 2 layouts for each profile (Event Coordinator) and (Admission Officer) for Customs Objects.

Daniel-Garcia-C commented:
I verify the profiles and layouts with the administrator profile, not any changes in "fields & relationships" in this commit.
